### PR TITLE
Agios 427.otp.crypto.demos

### PR DESCRIPTION
**Pull request by corinnekrych**
As discussed, some cleaning on ios-cookbook 1.6.x branch, I also apply template formatting for root README
@matzew mind to review?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,5 +24,5 @@ Podfile.lock
 Pods/
 
 #Docset/AppleDoc
+<<<<<<< HEAD
```
Comment (danielpassos) [inline .gitignore]: Seem a merge problem

**Comment by danielpassos** [comment]
@corinnekrych Could we rename the CryptoDemo by PasswordManager like [we did](https://github.com/aerogear/aerogear-android-cookbook/tree/master/PasswordManager) on Android land? wdyt?

**Comment by danielpassos** [comment]
Well, we also renamed the OTP example by [Two-Factor](https://github.com/aerogear/aerogear-android-cookbook/tree/master/Two-Factor)

**Comment by corinnekrych** [comment]
@danielpassos I renamed both apps as suggested. also clean-up README to apply template.

**Comment by cvasilak** [comment]
thank you @corinnekrych @danielpassos landed on https://github.com/aerogear/aerogear-ios-cookbook/commit/b74a2c249767e199683def29feb38fffe40fabb2

**Pull request merged**